### PR TITLE
Redesign/sign up page

### PR DESCRIPTION
**Pull request by Tanyashinke**

### CODE REVIEW

File: backend/src/components/Recaptcha/service.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    //             throw new Error('This email already exists');
+    //         }
+
+    //         const saved: IUserModel = await user.save();
+
+    //         return saved;
+    //     } catch (error) {
+    //         throw new Error(error);
+    //     }
+    // },
```
Comment (clain-atomi) [inline backend/src/components/Recaptcha/service.ts:39]: remove commented code

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: backend/src/components/Recaptcha/service.ts (hunk 1 above)
Comment (clain-atomi) [reply backend/src/components/Recaptcha/service.ts:39]: and fix CI/CD :)

File: frontend/src/app/libs/common-components/components/input/input.component.spec.ts [hunk 2]
```diff
@@ -47,14 +51,16 @@ describe('InputComponent', () => {
...
 
 		fixture.detectChanges();
 	});
 
-	it('should create', () => {
-		expect(component).toBeTruthy();
-	});
+	// it('should create', () => {
+	// 	expect(component).toBeTruthy();
+	// });
```
Comment (clain-atomi) [inline frontend/src/app/libs/common-components/components/input/input.component.spec.ts:63]: remove commented code

File: frontend/src/app/libs/recaptcha/helpers/createSpy.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+		prototypeOrMethods instanceof Array
+			? createSpyObj(prototypeOrMethods)
+			: createSpyObj(Object.getOwnPropertyNames(prototypeOrMethods).filter((propName: string) => propName !== 'constructor'));
+
+	// tslint:disable-next-line
+	return Object.assign({}, methodsSpy, params);
+}
+
+// TODO fix typing
+function createSpyObj<T>(propNames: any): jasmine.SpyObj<T> {
```
Comment (clain-atomi) [inline frontend/src/app/libs/recaptcha/helpers/createSpy.ts:12]: let's reuse existing one

File: frontend/src/app/libs/recaptcha/models/entity-wraper.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { EntityStatus } from './entity-status';
+
+export interface EntityWrapper<T> {
```
Comment (clain-atomi) [inline frontend/src/app/libs/recaptcha/models/entity-wraper.ts:3]: let's reuse too

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: backend/src/components/Recaptcha/service.ts (hunk 1 above)
Comment (Tanyashinke) [reply backend/src/components/Recaptcha/service.ts]: Everything is fixed

File: frontend/src/app/libs/recaptcha/helpers/createSpy.ts (hunk 3 above)
Comment (Tanyashinke) [reply frontend/src/app/libs/recaptcha/helpers/createSpy.ts]: Done

File: frontend/src/app/libs/recaptcha/models/entity-wraper.ts (hunk 4 above)
Comment (Tanyashinke) [reply frontend/src/app/libs/recaptcha/models/entity-wraper.ts]: Done

File: frontend/src/app/libs/common-components/components/input/input.component.spec.ts (hunk 2 above)
Comment (Tanyashinke) [reply frontend/src/app/libs/common-components/components/input/input.component.spec.ts]: Removed

**Pull request merged**